examples/beginner_diabetes.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2022 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beginner-friendly  usage example of TensorFlow Decision Forests (TF-DF).
+
+This example trains, display and evaluate a Random Forest model on the pima India's Diabetes dataset
```

Comment:
This example trains, displays and evaluates a Random Forest model on the Pima India's Diabetes dataset